config.example

```diff
@@ -8,5 +8,6 @@
     "nickserv_password": "yourpasswd",
     "auxchans": [],
     "extendedcommands": ["jaden","excuse","dong"],
-    "topmodifier": 0.05
+    "topmodifier": 0.05,
+    "repolink": "https://github.com/tonyravioli/dongerdong"
```

Comment:
Yeah but why have the repo link in the source? I think config is exactly what this type of thing is for.